test/simulation/TestSimulationManager.java

```diff
@@ -620,6 +620,109 @@ public void testPerformUntil() throws IOException {
         assertEquals(expectedCurrentState, sm.getCurrentStateString());
     }
 
+    @Test
+    public void testSetToInit() throws IOException {
```

Comment:
Please add a test that describes the behaviour when the following operations are performed in order:

1. `initializeWithModel`
2. `selectAlternate`
3. `performStep`
4. `init`